cookbooks/aws-parallelcluster-scheduler-plugin/files/default/compute_fleet_status/compute_fleet_status.py

```diff
@@ -0,0 +1,128 @@
+        raise Exception(f"Failed when updating fleet status with error: {e}")
+
+
+def get_dynamo_db_data(table):
+    try:
+        compute_fleet_item = table.get_item(ConsistentRead=True, Key={"Id": DB_KEY})
+        if not compute_fleet_item or "Item" not in compute_fleet_item:
+            raise Exception("COMPUTE_FLEET data not found in db table")
+        db_data = compute_fleet_item["Item"].get(DB_DATA)
+        return db_data
+
+    except Exception as e:
+        raise Exception(f"Failed when retrieving data from DynamoDB with error {e}.")
+
+
+def get_status_with_last_updated_time(table_name, region):
+    """Get compute fleet status and the last compute fleet status updated time."""
+    try:
+        table = boto3.resource("dynamodb", region_name=region).Table(table_name)
+        dynamo_db_data = get_dynamo_db_data(table)
+        print(
+            json.dumps(
+                {
+                    COMPUTE_FLEET_STATUS_ATTRIBUTE: dynamo_db_data.get(COMPUTE_FLEET_STATUS_ATTRIBUTE),
+                    COMPUTE_FLEET_LAST_UPDATED_TIME_ATTRIBUTE: dynamo_db_data.get(
+                        COMPUTE_FLEET_LAST_UPDATED_TIME_ATTRIBUTE
+                    ),
+                },
+                sort_keys=True,
+                indent=4,
+            )
+        )
+
+    except Exception as e:
+        raise Exception(f"Failed when retrieving fleet status from DynamoDB with error {e}.")
+
+
+def main():
+    try:
+        parser = argparse.ArgumentParser(description="Get or update compute fleet status of scheduler plugin.")
+        parser.add_argument(
+            "--table-name",
+            type=str,
+            required=True,
+            help="DynamoDB table name",
+        )
+        parser.add_argument(
+            "--region",
+            type=str,
+            required=True,
+            help="Region of cluster",
+        )
+        parser.add_argument(
+            "--status",
+            type=str,
+            required=False,
+            help="Specify the compute fleet status to set, can be PROTECTED",
+            choices={"PROTECTED"},
+        )
+        parser.add_argument(
+            "--action",
+            type=str,
+            required=True,
+            help="Get or update compute-fleet-status",
+            choices={"update", "get"},
+        )
+
+        args = parser.parse_args()
+        if args.action == "update" and not args.status:
+            parser.error("--status is required when 'action' is specified to 'update'.")
+        elif args.action == "get" and args.status:
+            parser.error("--status can not be specified when 'action' is 'get'.")
+
+        if args.action == "update":
+            update_status_with_last_updated_time(args.table_name, args.region, args.status)
+        else:
+            get_status_with_last_updated_time(args.table_name, args.region)
+    except Exception as e:
+        print(f"Failed to {args.action} compute fleet status, exception: {e}", file=sys.stderr)
```

Comment:
make format of error message consistent between the py and the bash, e.g. both with `ERROR:` prefix